src/main/app/src/views/toteutus/sections/ToteutuksenJarjestamistiedotSection.js

```diff
@@ -142,6 +166,26 @@ export class ToteutuksenJarjestamistiedotSection extends AbstractSection {
     }
   ];
 
+  //TODO: siirrä storeen
+  getLukuvuosiOptions = () => [
+    {
+      label: '2019',
+      key: '2019'
+    },
+    {
+      label: '2020',
+      key: '2020'
+    },
+    {
+      label: '2021',
+      key: '2021'
+    },
+    {
+      label: '2022',
+      key: '2022'
+    }
+  ];
+
   getLisattavaOsioActiveOptions = () => this.state.lisattavaOsioOptions.filter(entry => entry.active);
```

Comment:
Nämä arvot pitää laskea kuluvasta vuodesta. Niitä ei voi hard-codata. Toki ne voi nyt jättää näin, jos on kiire, mutta voisi ainakin laittaa TODO:n, että pitää korjata.

Reply:
Viittaa siis noihin vuosilukuihin. Kommentti tuli vähän hassuun paikkaan.

Reply:
Joo, päivitän TODO:t seuraavan bräntsiin. Nyt on tosiaan ilmeisesti kiire saada proto kuntoon, joten nyt ilmeisesti joudutaan oikomaan vähäsen.